pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "holland"
+version = "1.2.11"
+description = "Holland Core Plugins"
+entry_points="""
+    # Scripts generated by setuptools
+    [console_scripts]
+  	holland = holland.core.cmdshell:main
+
+  	# Holland subcommands
+  	[holland.commands]
+  	#help = holland.commands.help:Help
+  	listplugins = holland.commands.list_plugins:ListPlugins
```

Comment:
Should these help and restore sections be commented out?

Reply:
Help is done through argparser, and restore was experimental.